sdk/template/Azure.Template/src/TemplateClient.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Template.Models;
+
+namespace Azure.Template
+{
+    /// <summary> The Template service client. </summary>
```

Comment:
It might be worth calling out here or elsewhere in the file that this partial class implements the Convenience APIs for the DPG client.  You could even link back to where this is discussed in the getting-started documentation.  I think this would help service-team partners make this connection, and make it easier to understand the overall Azure SDK client development story.

Reply:
updated